config/enabled.yml

```diff
@@ -1004,6 +1004,10 @@ Style/WordArray:
   StyleGuide: '#percent-w'
   Enabled: true
 
+Style/YodaCondition:
+  Description: 'Use x == 5 instead of 5 == x.'
```

Comment:
I'd actually word this in more generic way - "don't use constants as the first operand of a comparison" or something like this.